src/lib.rs

```diff
@@ -1162,11 +1168,11 @@ fn handle_packet<H, W>(data: &[u8],
             Command::InsertAccessWatchpoint(wp) => {
                 handler.insert_access_watchpoint(wp).into()
             }
-            Command::RemoveSoftwareBreakpoint(addr, kind) => {
-                handler.remove_software_breakpoint(addr, kind).into()
+            Command::RemoveSoftwareBreakpoint(bp) => {
+                handler.remove_software_breakpoint(bp).into()
             }
-            Command::RemoveHardwareBreakpoint(addr, kind) => {
-                handler.remove_hardware_breakpoint(addr, kind).into()
+            Command::RemoveHardwareBreakpoint(bp) => {
+                handler.remove_hardware_breakpoint(bp).into()
```

Comment:
I'm starting to wonder if it wouldn't be worth writing a macro to wrap this repetitive pattern...